content/en/continuous_integration/tests/_index.md

```diff
@@ -22,6 +24,19 @@ further_reading:
 
 The [Tests][1] page, under the CI menu in Datadog, provides a test-first view into your CI health by showing you important metrics and results from your tests. It can help you investigate performance problems and test failures that concern you primarily because you work on the related code (and less because you maintain the pipelines they are run in).
 
+## Setup
+
+
+
+{{< whatsnext desc="Choose your language to set up tests visibility in Datadog:" >}}
+    {{< nextlink href="continuous_integration/tests/dotnet" >}}.NET{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/java" >}}Java{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/javascript" >}}JavaScript{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/python" >}}Python{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/ruby" >}}Ruby{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/swift" >}}Swift{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/junit_upload" >}}Uploading JUnit test report files to Datadog{{< /nextlink >}}
```

Comment:
This page is not a language, but there's no easy way to introduce this page either unless you added it as a For more information... type of sentence?

Reply:
Yeah, it's a framework for Java... so I considered bumping it higher in the list to be directly below Java.... but this is how it's organized in the left-nav so I left it. I'm ok with it just being the ugly duckling of this list and leaving it though because idk what would fit better